### DISCUSSION TITLE
Github Education Benefits not received Yet

### DISCUSSION THREAD

**Discussion by adityababhulkar1** [community/community#140671] [GitHub Education]
### Hi everyone,

I am writing to inquire about the status of my GitHub Education benefits. My application for these benefits was approved on 2nd Oct. However, as of today, I have not yet received access to the benefits.

I have verified that my student status is confirmed in my GitHub account. Could you please investigate this matter and provide an update on when I can expect my benefits to be activated?

Thank you for your prompt attention to this matter.

**Comment by davevad93** [comment]
Hi @adityababhulkar1 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @adityababhulkar1 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
